fixed mock data generation

daanonymous12 opened the pull request:
hard coded limits to stock price generations. Since we are not using monte carlo simulations and all price movements are based on only one path, probability of price approaching 0 or 1e52 is really high and need to be avoided.

daanonymous12 committed:
fixed mock data generation

hard coded limits to stock price generations. Since we are using monte carlo simulations and all price movements are based on ony one path, probability of price approaching 0 or 1e52 is really high and need to be avoided.

ttc290 commented on Mock_Data/mock_data_gen.py:
```diff
@@ -38,7 +38,9 @@ def simulate(self,start_date,end_date,interval):
                 temp = start_date
                 while temp <= start_date + timedelta(hours = 7, minutes = 30): # interval is # of times one wish to scan per hour
-                    cur_price = random.uniform(.98 * cur_price, 1.03 * cur_price) # 2% max change in price since aapl is large stock
+                    cur_price = (1 - (10 - random.randint(0,20))/100) * cur_price # 2% max change in price since aapl is large stock
```
You can probably write another method that takes a lower bound, an upper bound, and a price to generate the new price

daanonymous12 replied:
Fixed both issues, added parameters for adjusting lower and upper limit

ttc290 reviewed (commented):
L30-31 should be its own method as well
https://github.com/daanonymous12/auto_crawler/blob/4f5f9db825c25ddee3b7dc1ad23bde6c0ff763ca/Mock_Data/mock_data_gen.py#L30

daanonymous12 committed:
updated mock data

separated price generation and time split into separate function, included option to adjust upper and lower bound
